src/lib/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+# Some operating systems don't like compiling a library with
+# no source files
+
+#add_library(openstudiolib_static STATIC empty.cpp)
+
+#target_link_libraries(
+# openstudiolib_static
+# PUBLIC
+# openstudio_utilities
+# openstudio_airflow
+# openstudio_model
+# openstudio_energyplus
+# openstudio_measure
+# openstudio_osversion
+# openstudio_sdd
+# openstudio_isomodel
+# openstudio_gbxml
+# openstudio_radiance
+#)
+#
```

Comment:
That part is repeated. Remove the comments?

Reply:
done.